pkg/webhooks/machine_webhook.go

```diff
@@ -132,6 +135,7 @@ const (
 	azureCachingTypeNone               = "None"
 	azureCachingTypeReadOnly           = "ReadOnly"
 	azureCachingTypeReadWrite          = "ReadWrite"
+	azureRHCOSVersion                  = "412.86.202209220538-0"
```

Comment:
How often will this change?

Reply:
Hm, I have feeling that this might change over new z-stream releases...

Reply:
Every release AFAIK, we should try to find a way to keep this up to date by looking it up witihin the cluster or somewhere but for now I want to get things fixed.
FWIW installer update their config every release

Reply:
Is there any way to reference a `stable` RHCOS alias that points to the latest?

Reply:
@JoelSpeed WDYT about add a comment with some additional context?
I believe value of this constant came from this file: https://github.com/openshift/installer/blob/master/data/data/coreos/rhcos.json

Think future us might appreciate such breadcrumbs :)

Reply:
I already appreciate this breadcrumb :shipit: